### PR TITLE
Added a few new units and new prefixes for existing units.

### PR DESCRIPTION
**Pull request by DInozemtsev**
1. BritishThermalUnits: added new prefixes 'Kilo', 'Mega' and 'Giga' to support 'kBTU', 'mBTU', 'mmBTU'
2. Added a new unit 'CubicFootPerHour'
3. Pound: added new prefixes 'Kilo' and 'Mega' to support 'KLbs' and 'MLbs'
4. PoundsPerHour: added new prefixes 'Mega' to support 'Mlb/h'
5. Added a new unit 'ShortTonPerHour'
6. Added a new unit 'BritishThermalUnitPerHour'
7. CubicMeters: added new prefixes 'Hecto' and 'Kilo'
8. CubicFoot: added new prefixes 'Hecto', 'Kilo' and 'Mega'
9. ImperialGallons: added new prefixes 'Kilo' and 'Mega'
9. UsGallons: added new prefixes 'Kilo' and 'Mega'

### CODE REVIEW

**Comment by angularsen** [comment]
Awesome stuff, there is just a merge conflict you need to address. Please merge in latest from `origin/master` or rebase on top of it and fix the conflicts that arise, then push these new changes so the pull request is updated.

I think I saw you were a first-time github contributor, so if you are unsure about these things just ask and I'll help you out.

**Comment by DInozemtsev** [comment]
Hmm... I did merge from your master into mine and there was not any conflicts. However AppVeyor build fails with errors like this: 

`GeneratedCode\UnitClasses\Energy.g.cs(159,23): error CS0102: The type 'Energy' already contains a definition for 'DecathermsEc' [C:\projects\unitsnet\UnitsNet\UnitsNet.NetStandard10.csproj]`

Which is quite odd, because I can not find any duplication of `DecathermsEc` definition in `Energy.g.cs`. More than that, it builds perfectly on my local box. Am I missing something here? @angularsen could you please help me on this?

**Comment by angularsen** [comment]
I can check back in later today, but are you sure you merged in master from
angularsen repo and not your own fork/repo's master?

`git remote - v` to list remotes

`git branch - avv` to list all branches including remote branches

Assuming you cloned your own fork instead of the official repo you have to
add angularsen/UnitsNet as a remote then merge in changes from its master.

**Comment by DInozemtsev** [comment]
I've used following command to do merge into my branch:

`git pull https://github.com/angularsen/UnitsNet.git master`

And I can definitely see your latest commits in my master now.

**Comment by DInozemtsev** [comment]
Yay! Everything is green now. @angularsen thank you so much for pointing at the problem :)

**Comment by angularsen** [comment]
Awesome, good job :-)

File: UnitsNet.Tests/CustomCode/EnergyTests.cs [hunk 1]
```diff
@@ -28,8 +28,12 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double JoulesInOneJoule => 1;
 
+        protected override double KilobritishThermalUnitsInOneJoule => 9.4781339449889105832843629746176e-7;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/EnergyTests.cs]: I forgot to verify the latest units, I see a difference here with google (BTU, not kBTU): `9.47817...e-7`
https://www.google.no/search?q=britishThermalUnitsInOneJoule&oq=britishThermalUnitsInOneJoule&aqs=chrome..69i57.224j0j7&sourceid=chrome&ie=UTF-8

File: UnitsNet.Tests/CustomCode/VolumeTests.cs [hunk 2]
```diff
@@ -48,14 +48,24 @@ public class VolumeTests : VolumeTestsBase
...
+        protected override double HectocubicFeetInOneCubicMeter => 3.531472e-1;
+        protected override double HectocubicMetersInOneCubicMeter => 0.01;
 
         protected override double HectolitersInOneCubicMeter => 1E1;
 
         protected override double ImperialGallonsInOneCubicMeter => 219.96924;
 
         protected override double ImperialOuncesInOneCubicMeter => 35195.07972;
 
+        protected override double KilocubicFeetInOneCubicMeter => 3.531472e-2;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeTests.cs]: Wait, is this really a unit? Kilocubic feet? Sure this is not just 1000 cubicfeet? I can't find kilocubic feet on google.

File: UnitsNet.Tests/CustomCode/VolumeTests.cs [hunk 3]
```diff
@@ -48,14 +48,24 @@ public class VolumeTests : VolumeTestsBase
...
+        protected override double KilocubicMetersInOneCubicMeter => 0.001;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeTests.cs]: Same with kilocubic meters. Doesn't sound right?

File: UnitsNet.Tests/CustomCode/VolumeTests.cs [hunk 4]
```diff
@@ -48,14 +48,24 @@ public class VolumeTests : VolumeTestsBase
...
+        protected override double KiloimperialGallonsInOneCubicMeter => 2.1996924e-1;
+        protected override double KilousGallonsInOneCubicMeter => 2.6417217e-1;
+
         protected override double LitersInOneCubicMeter => 1E3;
+        protected override double MegacubicFeetInOneCubicMeter => 3.531472e-5;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeTests.cs]: And the megacubic ones too.

File: UnitsNet.Tests/CustomCode/VolumeTests.cs [hunk 5]
```diff
@@ -48,14 +48,24 @@ public class VolumeTests : VolumeTestsBase
...
+        protected override double MegaimperialGallonsInOneCubicMeter => 2.1996924e-4;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeTests.cs]: I do find megaimperial gallons though so this one should be OK, still sounds funny to me :-)

Comment (angularsen) [reply UnitsNet.Tests/CustomCode/VolumeTests.cs]: And that means mega US gallons probably also is OK.

File: UnitsNet.Tests/CustomCode/VolumeTests.cs (hunk 2 above)
Comment (DInozemtsev) [reply UnitsNet.Tests/CustomCode/VolumeTests.cs]: Hi @angularsen This is a unit of measure, that used in this product https://portfoliomanager.energystar.gov/ and called **kcf (thousand cubic feet)**
And, yeah, it sounds weird for me too, but guys in the States seems to be absolutely in love with some strange units of measure. Take it or leave it - it's up to you of course.

File: UnitsNet.Tests/CustomCode/VolumeTests.cs (hunk 4 above)
Comment (DInozemtsev) [reply UnitsNet.Tests/CustomCode/VolumeTests.cs]: The same here, they called it **Kcm (Thousand Cubic meters)**

File: UnitsNet.Tests/CustomCode/VolumeTests.cs (hunk 5 above)
Comment (DInozemtsev) [reply UnitsNet.Tests/CustomCode/VolumeTests.cs]: Looks like in U.S. they call them **KGal (thousand gallons) (US)** and **KGal (thousand gallons) (UK)**

File: UnitsNet.Tests/CustomCode/VolumeTests.cs (hunk 3 above)
Comment (DInozemtsev) [reply UnitsNet.Tests/CustomCode/VolumeTests.cs]: See above

**Pull request merged**